contest/contest_info.py

```diff
@@ -63,10 +63,38 @@ def get_running_contests():
 def get_contestant_list(contest):
```

Comment:
Maybe this function can be replaced by ```get_contestant(contest)```.
It needs a little modification in ```get_scoreboard(contest)``` function.